include/openssl/randerr.h

```diff
@@ -58,6 +59,7 @@ int ERR_load_RAND_strings(void);
 # define RAND_R_ERROR_ENTROPY_POOL_WAS_IGNORED            127
 # define RAND_R_ERROR_INITIALISING_DRBG                   107
 # define RAND_R_ERROR_INSTANTIATING_DRBG                  108
+# define RAND_R_RANDOM_DATA_WAS_IGNORED             137
```

Comment:
somehow the define does not line up with the surrounding lines?
and was the intention to name it RAND_R_ERROR_RANDOM_DATA_WAS_IGNORED ?